.travis.yml

```diff
@@ -1,7 +1,8 @@
 ---
 language: node_js
 node_js:
-  - "0.12"
+  - "4"
+  - "6"
```

Comment:
Can we also include `"stable"` here?

I'd also like to update the actual test command that gets runs from an individual ember-try scenario to use `ember try:each` instead. That way we don't eat up tons of Travis processes.